frontend/src/old-pages/Configure/Queues/SubnetMultiSelect.tsx

```diff
@@ -0,0 +1,44 @@
+import {Multiselect} from '@cloudscape-design/components'
+import React from 'react'
+import {useMemo} from 'react'
+import {useState} from '../../../store'
+import {subnetName} from '../util'
+import {Subnet} from './queues.types'
+
+function SubnetMultiSelect({value, onChange}: any) {
```

Comment:
todo: type this component! :) I know it's not doable for all the existing components, but I'd try to do this for everything we create from scratch.

Reply:
Ack, will add `SubnetMultiSelectProps` :)